src/app/views/authentication/profile/Profile.tsx

```diff
@@ -106,7 +106,7 @@ export class Profile extends Component<IProfileProps, IProfileState> {
     const persona: IPersonaSharedProps = {
       imageUrl: user.profileImageUrl,
       imageInitials: this.getInitials(user.displayName),
-      text: user.displayName,
+      text: user.displayName + ' ' + (this.props.permissionModeType ? messages['As user'] : messages['As Teams app']),
```

Comment:
```suggestion
      text: user.displayName + ' ' 
          + (this.props.permissionModeType 
              ? translateMessage('As user') 
              : translateMessage('As Teams app')),
```
This line felt annoying to read without any breaks;
Linebreaks for clarity, plus the change from `messages[]` to `translateMessage()`